tests/pds/pds_basic_test.go

```diff
@@ -69,6 +69,8 @@ func TestDataService(t *testing.T) {
 
 var _ = BeforeSuite(func() {
 	Step("get prerequisite params to run the pds tests", func() {
+		logrus.Infof("Initializing torpedo instance.")
+		InitInstance()
```

Comment:
Did you test this? Is it working? When I tried to do this there is a bug, https://portworx.atlassian.net/browse/PTX-13522

Reply:
It is working, please find the logs and URL to Jenkin's job in the description.